src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+import './_checkbox.scss';
+import { CheckboxProps } from '../../types/components/componentType';
+
+const Checkbox: React.FC<CheckboxProps> = ({
+  className,
+  classNameText,
+  checkboxText,
+  checked,
+  onChange,
+}) => {
+  const checkboxId =
+    'checkbox-' + checkboxText.toLowerCase().replace(/\s/g, '-');
+
+  return (
+    <div className="checkbox-container">
```

Comment:
It's a good practice to use more descriptive class names, especially if this component is part of a larger project. Or we will reuse those components and it's generic?